### ISSUE TITLE
Feature Request: Current values

### ISSUE THREAD

**Issue by gullitmiranda** [izelnakri/paper_trail#93]
what do you think about adding the possibility to register the pre-change in the versions?

with this feature we: 
1. Direct show changes instead of load all previous versions at find last change. Which is even more critical with multiple fields.
2. can search specific changes like: task status updated from TODO to DONE

i believe this feature can be added without changing the current lib behavior

**Comment by izelnakri** [comment]
Hi @gullitmiranda, the initial inspiration of this project the ruby `paper_trail` gem had ```item_from`` field, however this results in too much data being replicated. It is always possible to write custom queries and optimise them so on purpose I leave out the ```item_from``` field from this library. Let me know if this didn't answer your question, thanks for your request.

**Comment by gullitmiranda** [comment]
I agree if the item_from replicates the current value completely. But I don't see much of a problem in recording just the changed values.

In addition, this feature could be disabled by default, as is strict mode.

**Comment by izelnakri** [comment]
@gullitmiranda ok after some consideration, if you are willing to create a PR for this optional behavior, I think we can merge it after good tests for the cases of this feature when toggled and not.

**Comment by izelnakri** [comment]
Hi @gullitmiranda , I"m closing this issue since its been more than 4 years. Feel free to create a new issue or PR if you're still missing this feature.
